From 8e5fdffd52db3a940bbf2e2ce03c8faee166bc6a Mon Sep 17 00:00:00 2001
From: Francois Gaffie <francois.gaffie@intel.com>
Date: Tue, 30 Apr 2013 15:19:57 +0200
Subject: Add parameter key definition for BT

BZ: 47599
[CLEAN]

This patch adds both BT enable key define and BT value pairs define.

Change-Id: I661be8cac5e992a525c7e105e3eedc72be0d627a
Orig-Change-Id: I0d2b4b87e1dce2453b58b2a207b8e0797c0d24f2
Category: device enablement
Domain: AudioComms-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Francois Gaffie <francois.gaffie@intel.com>
---
 include/hardware/audio.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 2bc3aa3..f3dfc5e 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -82,6 +82,11 @@ __BEGIN_DECLS
  *  audio device parameters
  */
 
+/* BT chip state */
+#define AUDIO_PARAMETER_KEY_BLUETOOTH_STATE "bluetooth_enabled"
+#define AUDIO_PARAMETER_VALUE_BLUETOOTH_STATE_ON "true"
+#define AUDIO_PARAMETER_VALUE_BLUETOOTH_STATE_OFF "false"
+
 /* BT SCO Noise Reduction + Echo Cancellation parameters */
 #define AUDIO_PARAMETER_KEY_BT_NREC "bt_headset_nrec"
 #define AUDIO_PARAMETER_VALUE_ON "on"
-- 
1.7.4.1

